frame/alliance/src/migration.rs

```diff
@@ -51,22 +51,29 @@ mod v0_to_v1 {
 	use super::*;
 
 	pub fn migrate<T: Config<I>, I: 'static>() -> Weight {
-		if migration::clear_storage_prefix(
+		log::info!(target: LOG_TARGET, "Running migration v0_to_v1.");
+
+		let res = migration::clear_storage_prefix(
 			<Pallet<T, I>>::name().as_bytes(),
 			b"UpForKicking",
 			b"",
 			None,
 			None,
-		)
-		.maybe_cursor
-		.is_some()
-		{
+		);
+
+		if res.maybe_cursor.is_some() {
 			log::error!(
 				target: LOG_TARGET,
 				"Storage prefix 'UpForKicking' is not completely cleared."
 			);
 		}
 
-		T::DbWeight::get().writes(1)
+		log::info!(
+			target: LOG_TARGET,
+			"Cleared '{}' entries from 'UpForKicking' storage prefix",
+			res.unique
+		);
```

Comment:
nit: would be nicer if this info happened before the error. Cleared n then not completely cleared message is clearer than not completely cleared followed by a cleared n message.

Reply:
sweet (: make sense to me